config/production.json

```diff
@@ -1,4 +1,5 @@
 {
+	"$schema": "./default.schema.json",
```

Comment:
please remove all "$schema": "./default.schema.json" this is not a part of the feather config files.